### PR TITLE
Filter partitions

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request is to update the doc and create release note for the newly added filter partition feature.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[Release notes and docs update for partitions page change](https://sumologic.atlassian.net/browse/SUMO-238438)

### CODE REVIEW

File: blog-service/2024-04-02-filter-partitions.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  - accounts
+  - filter partitions
+  - release note
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+New - We're happy to introduce you to new partitions page enhancement, which allows you to filter the partitions based on the type of partitions - **User-Defined Partitions** and **System Partitions**. [Learn more](/docs/manage/partitions/run-search-against-partition/#filter-and-search-a-partition).
```
Comment (jpipkin1) [inline blog-service/2024-04-02-filter-partitions.md:14]:
```suggestion
We're happy to introduce a new partitions page enhancement which allows you to filter the partitions based on the type of partitions, either **User-Defined Partitions** or **System Partitions**. [Learn more](/docs/manage/partitions/run-search-against-partition/#filter-and-search-a-partition).
```

File: docs/manage/partitions/run-search-against-partition.md [hunk 2]
```diff
@@ -17,6 +17,15 @@ To search a particular partition, specify the `_index` metadata field with the n
 You can only use `_index` in the keyword search expression that scopes the search, in other words, before the first pipe (`|`) in the search.
 :::
 
+## Filter and search a partition
+
+Filter the partition depending on the type of partition using the dropdown provided above the table. Accordingly, you can view the overall allocated storage details with the percentage of storage used for the selected partition type. You can select between two type of partitions:
+
+- **User-Defined Partitions**. Type of partitions which are created by the user.
+- **System Partitions**. Type of partitions which are created from sumo to optimize the performance of the query.
```
Comment (jpipkin1) [inline docs/manage/partitions/run-search-against-partition.md:25]:
```suggestion
- **System Partitions**. Type of partitions which are created from Sumo Logic to optimize the performance of the query.
```

**Review by jpipkin1 — APPROVED** [review]

File: blog-service/2024-04-02-filter-partitions.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  - accounts
+  - filter partitions
+  - release note
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're happy to introduce a new partitions page enhancement which allows you to filter the partitions based on the type of partitions, either **User-Defined Partitions** or **System Partitions**. [Learn more](/docs/manage/partitions/run-search-against-partition/#filter-and-search-a-partition).
```
Comment (kimsauce) [inline blog-service/2024-04-02-filter-partitions.md:14]:
```suggestion
We're happy to introduce a new partitions page enhancement that allows you to filter the partitions based on the type of partitions, either **User-Defined Partitions** or **System Partitions**. [Learn more](/docs/manage/partitions/run-search-against-partition/#filter-and-search-a-partition).
```

File: docs/manage/partitions/run-search-against-partition.md [hunk 4]
```diff
@@ -17,6 +17,15 @@ To search a particular partition, specify the `_index` metadata field with the n
 You can only use `_index` in the keyword search expression that scopes the search, in other words, before the first pipe (`|`) in the search.
 :::
 
+## Filter and search a partition
+
+Filter the partition depending on the type of partition using the dropdown provided above the table. Accordingly, you can view the overall allocated storage details with the percentage of storage used for the selected partition type. You can select between two type of partitions:
```
Comment (kimsauce) [inline docs/manage/partitions/run-search-against-partition.md:22]:
```suggestion
You'll need to filter the partition, depending on the type of partition using the dropdown provided above the table. Accordingly, you can view the overall allocated storage details with the percentage of storage used for the selected partition type. You can select between two type of partitions:
```

**Review by kimsauce — APPROVED** [review]
two minor suggestions

**Pull request merged**